draft-ietf-masque-connect-udp.md

```diff
@@ -526,6 +526,8 @@ to arbitrary targets, as that could allow bad actors to send traffic and have it
 attributed to the UDP proxy. HTTP servers that support UDP proxying ought to
 restrict its use to authenticated users.
 
+UDP proxies have similar properties to TCP proxies when it comes to facilitating
+denial of service attacks, even though TCP offers better protection in theory.
```

Comment:
What is the theory here?  Tastefulness of TCP flows?  Maybe call out what the UDP deficiencies are and how it might impact proxying?  Just a friendly suggestion here.

Reply:
That's what this paragraph is attempting to do but I'm sure it can be improved, do you have a specific suggestion in mind?

Reply:
```suggestion
denial of service attacks. In theory, the stateful nature of TCP provides better protection that stateless UDP. However, in practice there are negligible differences when considering proxy.
```